tests/util/translator/parse/common/parse-base-schema.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import { EElementType, IPrefillAttributes } from "src/context-providers";
+import { parseBaseSchema } from "src/translator/parse/common";
+import { TElementSchema } from "src/translator/parse/types";
+import { generateMockElement, generateMockElementSchema } from "../../helper";
+
+describe("ParseBaseSchema", () => {
+    beforeEach(() => {
+        global.ResizeObserver = jest.fn().mockImplementation(() => ({
+            observe: jest.fn(),
+            unobserve: jest.fn(),
+            disconnect: jest.fn(),
+        }));
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+        jest.resetAllMocks();
+    });
+
+    it("should parse the base schema and return the base attributes of an element", () => {
+        const MOCK_SCHEMA = generateMockElementSchema({
```

Comment:
just noticed the naming, we only use UPPER_SNAKE_CASE for const declared outside functions, this way we know that they are never meant to be mutated. for const declared within functions, you can use the regular camelCase.

Reply:
Got it. Have renamed those applicable ones related to parsing for this PR. For the rest, I will gradually edit them in future PRs.